doc/rst/language/spec/variables.rst

```diff
@@ -229,39 +267,97 @@ initialization, not assignment, of that variable.
       5
 
 
-Split initialization does not apply:
-
- * when the variable is a field, config variable, or ``extern`` variable.
- * when an applicable assignment statement setting the variable could not
-   be identified
- * when an applicable assignment statement is in one branch of a
-   conditional but not in the other, unless:
+   *Example (split-init-select-dce.chpl)*
```

Comment:
dce here in the filename and later in the source code is jargon-y. Let's find a different way to say it that fits more with the terminology in the spec. (Note that historically I think we have called this "param folding", but I don't think the spec even uses that term). IMO the mentions of DCE here are more a name-drop than fundamental, so you could use something more descriptive in the text and simply remove DCE from the file name / routine name.

Reply:
Sounds good. I went with `split-init-select-ignored-path.chpl` and `splitInitsBecausePathKnown`.